web/packages/teleterm/src/ui/services/workspacesService/accessRequestsService.ts

```diff
@@ -99,34 +99,40 @@ export class AccessRequestsService {
     });
   }
 
-  async addOrRemoveKubeNamespaces(namespaceUris: KubeResourceNamespaceUri[]) {
+  async updateNamespacesForKubeCluster(
+    namespaceUris: KubeResourceNamespaceUri[],
+    kubeClusterUri: string
```

Comment:
So we need `kubeClusterUri` to validate if all namespaceUris come from the same kube cluster? 
It feels to me too complicated, perhaps we shouldn't store namespace uris but just their names.
Since a namespace is always kept in a specific kube cluster, we don't need its whole uri.
So it would be:
```ts
async updateNamespacesForKubeCluster(
    kubeUri: KubeUri
    namespaces: string[],
)
```

Reply:
>If we stored sole namespace names, we'd have to always remember about converting them to URIs when taking them out of AccessRequestsService.

Right, converting to URI could be inconvenient. Well, I don't have a strong opinion on this, so let's keep it as is.